WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -670,19 +633,26 @@ extension StorePickerViewController: UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
-        guard state.multipleStoresAvailable else {
+        guard viewModel.multipleStoresAvailable else {
             // If we only have a single store available, don't allow the row to be selected
             return false
         }
         return true
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        guard let site = state.site(at: indexPath) else {
+        guard let site = viewModel.site(at: indexPath) else {
             tableView.deselectRow(at: indexPath, animated: true)
             return
         }
 
+        guard site.isWooCommerceActive else {
+            ServiceLocator.analytics.track(.sitePickerNonWooSiteTapped, withProperties: ["is_non_atomic": !site.isJetpackConnected])
```

Comment:
`is_non_atomic` feels a bit confusing to me, because self-hosted sites are also not atomic 🤔 if I understand correctly, this event property is to know which stores are eligible for Woo/plugin installation like in `NoWooErrorViewModel.showsInstallButton` below? Would another name like `can_install_woo` be more clear, and an extension like `Site.isEligibleForWooInstallation` be helpful?

I'm also just curious how we determine whether a site is eligible for Woo/plugin installation. It looks like the requirement is `site.isJetpackConnected && site.isJetpackThePluginInstalled` below, does it account for user roles that cannot install plugins like shop managers? I'm sure these have been considered, I'd just like to know more since the code just checks for the site's Jetpack status here 😄

Reply:
The idea is to check for the number of users trying to use simple sites here (WP.com site without Business or E-commerce plans). For HEs the term simple site is more common, but I'm not sure if that's the case for us as well.

> I'm also just curious how we determine whether a site is eligible for Woo/plugin installation

This is a good idea! I'm thinking about adding this for the next improvements of the Woo installation flow. Maybe in the next PR.

Reply:
In fact I think we don't need to check for eligibility, since the web view handles that already:

<img src="https://user-images.githubusercontent.com/5533851/184085667-2bbada2f-bb98-4299-b85c-b0fcfa21a840.png" width=320 />